MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -18,6 +18,11 @@ public enum OfflineRoutingError: LocalizedError {
      */
     case standard(DirectionsError)
     
+    /**
+     The router did not finish the request
+     */
+    case cancelled
```

Comment:
Amazingly, Apple can’t decide whether it should be “[canceled](https://developer.apple.com/search/?q=canceled)” or “[cancelled](https://developer.apple.com/search/?q=cancelled)”. Either is fine here. 🤷‍♂️